crates/bevy_ecs/src/core/filter.rs

```diff
@@ -31,12 +31,52 @@ pub struct Or<T>(pub T);
 
 /// Query transformer that retrieves components of type `T` that have been mutated since the start of the frame.
 /// Added components do not count as mutated.
+/// This filter is useful as a performance optimization as it means that a system can be made to run only when
+/// it needs to rather than every frame.
+///
+/// Because the ordering of systems can change and this filter is only effective on changes before the query executes
+/// you need to use explicit dependency ordering or ordered stages for these query filters to be useful.
+///
+/// Example:
+/// ```
+/// use bevy::prelude::*;
+///
+/// fn print_moving_objects_system(query: Query<(Name,), (Mutated<Transform>,)>) {
```

Comment:
no need to wrap the types in tuples, you can have
```rust
fn print_moving_objects_system(query: Query<Name, Mutated<Transform>>) {
    for name in query.iter() {
        println!("Entity Moved: {}", name)
    }
}
```

Reply:
Cool! I didn't know that, I've been putting all my queries in tuples.......